docs/docs-book/src/using-coffee.md

```diff
@@ -3,44 +3,111 @@
 Coffee is a plugin manager and development tool for core lightning nodes. It helps automate configuration and installation of plugins, and provides plugin templates for all the official languages.
 
 Coffee is a command line utility that provides access to a wide range of tasks.
+
 ## First Configuration
 
+> ✅ Implemented
+
+Coffe is able to manage the configuration of your core lightning and all the plugins connected with it, so
+in order to use coffe the user need to point coffee in the current home of core lightning.
+
+By default the core lightning home is stored in the `/home/<user>/.lightnig`, and you can do it with the
+following command
+
+```bash
+coffee setup /home/alice/.lightning
+```
+
+Then you will fine an include at the end of a config file at `/home/alice/.lightnig/bitcoin/config`, 
```

Comment:
```suggestion
Then you will find an include at the end of a config file at `/home/alice/.lightning/bitcoin/config`, 
```

Reply:
note the `/home/alice` in the previous example, so we keep it consistent :)